web/public/index.html

```diff
@@ -82,7 +82,7 @@
         }
     </style>
 
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
+    <link href="assets/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
```

Comment:
ideally this would be pulled from npm rather than stored locally

Reply:
I have no idea on how to integrate this, sorry. I have seen the newest version is 4.3.1 here https://www.npmjs.com/package/bootstrap, worth updating too?

Reply:
usually it means using `node_modules` in a way that webpack likes

Reply:
I've solved this part by upgrading a bunch of dependencies. The font stuff is all that's left I think